src/main/scala/sbtecr/EcrPlugin.scala

```diff
@@ -7,6 +7,8 @@ import sbt._
 import scala.language.postfixOps
 
 object EcrPlugin extends AutoPlugin {
+  
+  lazy val version:SettingKey[String] = settingKey[String]("The ECR docker image version.")
```

Comment:
version has to be outside autoImport because it is already imported into build.sbt by sbt-native-packager.  There is a common namespace for setting keys across multiple plugins for a given project.

If we were to put it inside the import statement it would throw an error saying something like the setting key "version" is imported twice. In order to put it inside autoImport the name would need to be changed to something like ecrVersion.